app/models/Document.js

```diff
@@ -73,6 +75,14 @@ export default class Document extends BaseModel {
     return !!this.lastViewedAt && this.lastViewedAt < this.updatedAt;
   }
 
+  @computed
+  get new(): boolean {
```

Comment:
```suggestion
  get isNew(): boolean {
```

Suggest `isNew` to match other computed booleans and also not step on the reserved `new` keyword.